R/tm_g_ae_oview.R

```diff
@@ -372,7 +368,6 @@ srv_g_ae_oview <- function(id,
         card <- teal.reporter::TealReportCard$new()
         card$set_name("AE Overview")
         card$append_text("AE Overview", "header2")
-        card$append_text("Filter State", "header3")
```

Comment:
Did we forget to append the source code in osprey? I'm not seeing card$append_src here which I do see in teal.goshawk